config/routes.rb

```diff
@@ -81,6 +81,8 @@
   # for now dashboard path and root path are same.
   get "error_401" => "pages#unauthorized"
   get "dashboard" => "dashboards#index"
+  get "manager" => "dashboards#index_manager"
```

Comment:
skip this route, I will add the necessary changes to make this work automatically

Reply:
removed the path